test_scripts/Smoke/commonSmokeApi.lua

```diff
@@ -0,0 +1,245 @@
+            RAISE_EVENT(event, event, "PTU event")
+            checkIfPTSIsSentAsBinary(d2.binaryData)
+            local corIdSystemRequest = mobileSession:SendRPC("SystemRequest", { requestType = "PROPRIETARY", fileName = policy_file_name }, ptu_file_name)
+            EXPECT_HMICALL("BasicCommunication.SystemRequest")
+            :Do(function(_, d3)
+                self.hmiConnection:SendResponse(d3.id, "BasicCommunication.SystemRequest", "SUCCESS", { })
+                self.hmiConnection:SendNotification("SDL.OnReceivedPolicyUpdate",
+                  { policyfile = policy_file_path .. "/" .. policy_file_name })
+              end)
+            mobileSession:ExpectResponse(corIdSystemRequest, { success = true, resultCode = "SUCCESS" })
+            :Do(function() os.remove(ptu_file_name) end)
+          end)
+        :Times(AtMost(1))
+      end
+    end)
+end
+
+--[[Module functions]]
+
+function commonSmokeApi.preconditions()
+  commonFunctions:SDLForceStop()
+  commonSteps:DeletePolicyTable()
+  commonSteps:DeleteLogsFiles()
+end
+
+function commonSmokeApi.getDeviceMAC()
+  return config.deviceMAC
+end
+
+function commonSmokeApi.getPathToSDL()
+  return config.pathToSDL
+end
+
+function commonSmokeApi.getMobileAppId(pAppId)
+  if not pAppId then pAppId = 1 end
+  return config["application" .. pAppId].registerAppInterfaceParams.appID
+end
+
+function commonSmokeApi.getSelfAndParams(...)
+  local out = { }
+  local selfIdx = nil
+  for i,v in pairs({...}) do
+    if type(v) == "table" and v.isTest then
+      table.insert(out, v)
+      selfIdx = i
+      break
+    end
+  end
+  local idx = 2
+  for i = 1, table.maxn({...}) do
+    if i ~= selfIdx then
+      out[idx] = ({...})[i]
+      idx = idx + 1
+    end
+  end
+  return table.unpack(out, 1, table.maxn(out))
+end
+
+function commonSmokeApi.getMobileSession(pAppId, self)
+  self, pAppId = commonSmokeApi.getSelfAndParams(pAppId, self)
+  if not pAppId then pAppId = 1 end
+  return self["mobileSession" .. pAppId]
+end
+
+function commonSmokeApi.activateApp(pAppId, self)
+  self, pAppId = commonSmokeApi.getSelfAndParams(pAppId, self)
+  if not pAppId then pAppId = 1 end
+  local pHMIAppId = hmiAppIds[config["application" .. pAppId].registerAppInterfaceParams.appID]
+  local mobSession = commonSmokeApi.getMobileSession(pAppId, self)
+  local requestId = self.hmiConnection:SendRequest("SDL.ActivateApp", { appID = pHMIAppId })
+  EXPECT_HMIRESPONSE(requestId)
+  mobSession:ExpectNotification("OnHMIStatus",
+    {hmiLevel = "FULL", audioStreamingState = "AUDIBLE", systemContext = "MAIN"})
+  commonTestCases:DelayedExp(commonSmokeApi.minTimeout)
+end
+
+function commonSmokeApi.start(pHMIParams, self)
+  self, pHMIParams = commonSmokeApi.getSelfAndParams(pHMIParams, self)
+  self:runSDL()
+  commonFunctions:waitForSDLStart(self)
+  :Do(function()
+      self:initHMI(self)
+      :Do(function()
+          commonFunctions:userPrint(35, "HMI initialized")
```

Comment:
@AKalinich-Luxoft Use user_modules/consts for color

Reply:
@AByzhynar thanks, will use it. See https://github.com/smartdevicelink/sdl_atf_test_scripts/pull/1697/commits/1f9a96a07f40b1051bd1cd0e5a3b77a979c40def